nebula3/data/DataObject.py

```diff
@@ -24,6 +24,23 @@
     Time,
 )
 
+__AS_MAP__ = {
```

Comment:
why use `Double underscor` as for this name here?

Reply:
In my point of view, I want users to call `cast()` instead of handling by themselves. Thus, I use this way to prevent users from importing `AS_MAP`. If they really know what they are doing, then they should construct a map and casting by themselves.

Reply:
make sense ~